cli/cli/commands/service/start/start.go

```diff
@@ -0,0 +1,122 @@
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/services"
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/kurtosis_engine_rpc_api_bindings"
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/lib/kurtosis_context"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/highlevel/enclave_id_arg"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/highlevel/engine_consuming_kurtosis_command"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/highlevel/service_identifier_arg"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/lowlevel/args"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_framework/lowlevel/flags"
+	"github.com/kurtosis-tech/kurtosis/cli/cli/command_str_consts"
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface"
+	metrics_client "github.com/kurtosis-tech/metrics-library/golang/lib/client"
+	"github.com/kurtosis-tech/stacktrace"
+)
+
+const (
+	enclaveIdentifierArgKey = "enclave"
+	isEnclaveIdArgOptional  = false
+	isEnclaveIdArgGreedy    = false
+
+	serviceIdentifierArgKey        = "service"
+	isServiceIdentifierArgOptional = false
+	isServiceIdentifierArgGreedy   = false
+
+	kurtosisBackendCtxKey = "kurtosis-backend"
+	engineClientCtxKey    = "engine-client"
+
+	starlarkScript = `
+def run(plan, args):
+	plan.start_service(name=args["service_name"])
+`
+	doNotDryRun        = false
+	defaultParallelism = 4
+
+	useDefaultMainFile = ""
+)
+
+var ServiceStartCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisCommand{
+	CommandStr:                command_str_consts.ServiceStartCmdStr,
+	ShortDescription:          "Restarts a stopped service",
+	LongDescription:           "Restarts the temporarily stopped service with the given service identifier in the given enclave",
+	KurtosisBackendContextKey: kurtosisBackendCtxKey,
+	EngineClientContextKey:    engineClientCtxKey,
+	Args: []*args.ArgConfig{
+		enclave_id_arg.NewEnclaveIdentifierArg(
+			enclaveIdentifierArgKey,
+			engineClientCtxKey,
+			isEnclaveIdArgOptional,
+			isEnclaveIdArgGreedy,
+		),
+		service_identifier_arg.NewServiceIdentifierArg(
+			serviceIdentifierArgKey,
+			isServiceIdentifierArgGreedy,
+			isServiceIdentifierArgOptional,
+		),
+	},
+	Flags:   []*flags.FlagConfig{},
+	RunFunc: run,
+}
+
+func run(
+	ctx context.Context,
+	_ backend_interface.KurtosisBackend,
+	_ kurtosis_engine_rpc_api_bindings.EngineServiceClient,
+	_ metrics_client.MetricsClient,
+	_ *flags.ParsedFlags,
+	args *args.ParsedArgs,
+) error {
+	enclaveIdentifier, err := args.GetNonGreedyArg(enclaveIdentifierArgKey)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred getting the enclave identifier value using key '%v'", enclaveIdentifierArgKey)
+	}
+
+	serviceIdentifier, err := args.GetNonGreedyArg(serviceIdentifierArgKey)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred getting the service ID value using key '%v'", serviceIdentifierArgKey)
```

Comment:
nit: "service identifier" in place of "service ID" to be consistent with the "enclave identifier" wording above